driver/src/bin/stablex.rs

```diff
@@ -0,0 +1,23 @@
+#[macro_use]
+extern crate log;
+extern crate simple_logger;
+
+use driver::contracts::stablex_contract::StableXContractImpl;
+use driver::driver::stablex_driver::StableXDriver;
+
+use std::thread;
+use std::time::Duration;
+
+fn main() {
+    simple_logger::init_with_level(log::Level::Info).unwrap();
+
+    let contract = StableXContractImpl::new().unwrap();
+    let mut price_finder = driver::util::create_price_finder();
```

Comment:
nit: the past two lines might be in a different level of coupling with the actual implementation. Line 14 instantiates a specific implementation, the 15 calls a factory method that instantiates the specific implementation.

Reply:
Do you mean, the two lines together have a different level of coupling than the rest or between each other there is a different level of coupling?

I can see the first but not really the second case, since both lines "wire" up the implementation that needs to be used (one is using a helper method since it is somewhat more involved).

Are you suggesting to pass the env var into the `create_price_finder()` method in order to not rely on static state?

Reply:
I mean that stablex.rs knows the implementation details in line 14, but in line 15 not.

It would make sense to me to also use a factory for line 14 and have stablex.rs depend only on an interface. This is nit/ubernit right now